### ISSUE TITLE
Support Periodic Notes v1.0.0-beta

### ISSUE THREAD

**Issue by profucius** [jamiefdhurst/obsidian-auto-periodic-notes#20]
Hi there, I'd like to try your plugin in Obsidian. However when I install and enable it, I only get this message, and the plugin does not load. Any thoughts on how I can proceed?

![image](https://github.com/user-attachments/assets/9de49ed7-92c3-44b0-b1c5-3ef831d3a1c0)

**Comment by jamiefdhurst** [comment]
Thanks for submitting your issue. Just need to ask a few questions to see how I can recreate and check:

- which version of Obsidian are you using and on which platform?
- which other plugins do you have installed and enabled?

Can you also check in the Developer Tools console to see exactly what it says in there?

Then I can see if I can recreate and identify the issue.

**Comment by profucius** [comment]
Thanks for the reply. How can I access the Developer Tools console within Obsidian?

**Comment by jamiefdhurst** [comment]
On a Mac, use the View menu and select Toggle Developer Tools. That will
bring up a console that will show any errors that may have occurred.

**Comment by profucius** [comment]
I'm on Windows, doesn't include a menu bar like the Mac does. I found that Ctrl+Shirt+I shows the dev console.

I reproduced the issue and grabbed this from the Console pane:

```
Auto Periodic Notes: styles.css not found
```
```
app.js:1 Plugin failure: auto-periodic-notes TypeError: Cannot read properties of undefined (reading 'enabled')
    at PeriodicNotes.convertPeriodicNotesSettings (plugin:auto-periodic-notes:820:60)
    at AutoPeriodicNotes.syncPeriodicNotesSettings (plugin:auto-periodic-notes:930:44)
    at AutoPeriodicNotes.onLayoutReady (plugin:auto-periodic-notes:912:10)
    at t.onLayoutReady (app.js:1:2276057)
    at AutoPeriodicNotes.onload (plugin:auto-periodic-notes:900:24)
```

**Comment by jamiefdhurst** [comment]
Thank you! That's extremely helpful, I'll do my best to get this fixed.

**Comment by jamiefdhurst** [comment]
@profucius it seems the Periodic Notes settings aren't loading correctly, which is odd. Which version of the Periodic Notes plugin do you have enabled? Could you screenshot the community plugins page showing it, like this?

<img width="622" alt="Screenshot 2024-09-28 at 08 00 07" src="https://github.com/user-attachments/assets/fb1061a6-4363-453f-b24f-fd39a957b03e">

**Comment by profucius** [comment]
I'm updated to the latest version that Obsidian allowed me to install:

![image](https://github.com/user-attachments/assets/3870b543-a0a0-4c57-94da-c2337a286722)

Perhaps because I'm on a beta? I'm not sure how I have that installed but you don't? Is there an obsidian setting that allows for installing beta plugin releases that I'm not seeing?

**Comment by jamiefdhurst** [comment]
Yes, I suspect this is the beta version that's causing the discrepency. I suspect I can try and test this a little better, so thank you!

**Comment by jamiefdhurst** [comment]
@profucius The settings are just the tip of the ioceberg in terms of changes, so there's going to need to be a bit more work as part of this to ensure the plugin works fully with Periodic Notes v1.0.0. I'll continue to work on it in this issue, but it's not a simple fix so may take me a little longer as I find the time.

**Comment by profucius** [comment]
No worries, I'm glad we at least caught the reason. Still strange how the beta wasn't available to you as it was to me, seeing it's age, but perhaps there is a setting in obsidian that prevented the beta from appearing to you. Take your time!

**Comment by Azurewren** [comment]
Has there been any progress on this. I am using **Auto Periodic Notes** version 0.2.3, and **Periodic Notes** version 1.0.0-beta.3 and am having the same issue. I get the 'Failed to load' message when I enable this plugin.
